client/src/components/InlinePanel/index.js

```diff
@@ -209,6 +209,36 @@ export class InlinePanel extends ExpandingFormset {
     );
   }
 
+  addTabIndex(prefix) {
+    /*
+    add tabindex -1 into newly created form if attr not present and 
+    remove attr from old forms on blur event, if not present previously,
+    */
```

Comment:
Move this doc description to above the method and start it with `/**`. This will make it a JSDoc string.